RELEASE.md

```diff
@@ -4,21 +4,24 @@
 
 To release a version _“v0.W.Z”_ of Kuadrant Operator in GitHub and Quay.io, follow these steps:
 
-1. Kuadrant dependencies need to be released first:
+1. Verify correct versions for the Kuadrant release and its dependencies are listed in the `release.toml` file.
+   The verification should be done as part of the community call prior to cutting the release.
+   Update versions if required, release of dependency maybe required, see step 2.
+   This file is located in the root of the kuadrant operator.
+
+2. (Optional) Release Kuadrant dependencies:
    * [Authorino Operator](https://github.com/Kuadrant/authorino-operator/blob/main/RELEASE.md).
    * [Limitador Operator](https://github.com/Kuadrant/limitador-operator/blob/main/RELEASE.md).
    * [DNS Operator](https://github.com/Kuadrant/dns-operator/blob/main/docs/RELEASE.md).
    * [WASM Shim](https://github.com/Kuadrant/wasm-shim/).
    * [Console Plugin](https://github.com/Kuadrant/kuadrant-console-plugin).
 
-2. Run the GHA [Release operator](https://github.com/Kuadrant/kuadrant-operator/actions/workflows/release.yaml); make
-   sure to fill all the fields:
+   Once dependencies have being released jump to step 1.
 
+3. Run the GHA [Release operator](https://github.com/Kuadrant/kuadrant-operator/actions/workflows/release.yaml); make
+   sure to fill all the fields:
     * Branch containing the release workflow file – default: `main`
     * Commit SHA or branch name of the operator to release – usually: `main`
-    * Operator version to release (without prefix) – i.e. `0.W.Z`
-    * Kuadrant dependencies (WASM Shim, Console Plugin, Authorino, Limitador and DNS operators) versions (without prefix) – i.e. `0.X.Y`
-    * If the release is a prerelease
```

Comment:
This might still be useful for RCs

Reply:
I don't think it does. It is a choice that whom ever is doing the release would have to make at time of running the release workflow. This choice introduces a chance to make a mistake, and we want to reduce that chance.

I would expect that in the `release.toml` file when stating the version, it would be at that point the version is marked as a pre-release. This can be reviewed by a second set of eyes, reducing the chance for errors.

Reply:
sure ok makes sense that we can infer from the release file